pkg/odo/genericclioptions/clientset/clientset.go

```diff
@@ -139,7 +139,7 @@ func Fetch(command *cobra.Command) (*Clientset, error) {
 		dep.WatchClient = watch.NewWatchClient(dep.DeleteClient)
 	}
 	if isDefined(command, DEV) {
-		dep.DevClient = dev.NewDevClient(dep.WatchClient, dep.KubernetesClient)
+		dep.DevClient = dev.NewDevClient(dep.WatchClient)
```

Comment:
Please also remove dependency in `subdeps`

Reply:
Done.